config/routes.rb

```diff
@@ -18,6 +18,7 @@
     namespace :v0 do
       resources :tokens, only: [:create]
       get 'manifests/search', to: 'manifests#search'
+      post 'manifests/validate', to: 'manifests#validate'
```

Comment:
I see your point about validation being a verb not a noun. I think it can go either way, but fine to keep as-is. 

What does the 1 represent at `POST /manifests?validate=1` ?